src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/AccountController.cs

```diff
@@ -82,6 +83,19 @@ public async Task<IActionResult> GetProviderPaymentsPriority(long accountId)
             return Ok(response);
         }
 
+        [HttpGet]
+        [Route("{employerAccountId}/employer-account-summary")]
```

Comment:
`{accountId}` for consistency?

Reply:
could the route just be `/summary` as we are already in the context of an employer account?

Reply:
done